src/utils.ts

```diff
@@ -113,29 +116,46 @@ export async function fetchFlutterServerPort({
    portReleaseCallback,
    packageName,
    flutterCaps,
+   isIosSimulator,
 }: {
    udid: string;
    systemPort?: number | null;
    portForwardCallback?: PortForwardCallback;
    portReleaseCallback?: PortReleaseCallback;
    packageName: string;
    flutterCaps: DriverCaps<FlutterDriverConstraints>;
+   isIosSimulator: boolean;
```

Comment:
I am not sure this flag is needed now since we have the driver instance (`this` context here)